astropy/io/fits/fitstime.py

```diff
@@ -32,30 +32,30 @@
 
 
 # Global time reference coordinate keywords
+OBSGEO_XYZ = ("OBSGEO-X", "OBSGEO-Y", "OBSGEO-Z")
+OBSGEO_LBH = ("OBSGEO-L", "OBSGEO-B", "OBSGEO-H")
 TIME_KEYWORDS = (
-    "TIMESYS",
-    "MJDREF",
-    "JDREF",
-    "DATEREF",
-    "TREFPOS",
-    "TREFDIR",
-    "TIMEUNIT",
-    "TIMEOFFS",
-    "OBSGEO-X",
-    "OBSGEO-Y",
-    "OBSGEO-Z",
-    "OBSGEO-L",
-    "OBSGEO-B",
-    "OBSGEO-H",
-    "DATE",
-    "DATE-OBS",
-    "DATE-AVG",
-    "DATE-BEG",
-    "DATE-END",
-    "MJD-OBS",
-    "MJD-AVG",
-    "MJD-BEG",
-    "MJD-END",
+    (
+        "TIMESYS",
+        "MJDREF",
+        "JDREF",
+        "DATEREF",
+        "TREFPOS",
+        "TREFDIR",
+        "TIMEUNIT",
+        "TIMEOFFS",
+        "DATE",
+        "DATE-OBS",
+        "DATE-AVG",
+        "DATE-BEG",
+        "DATE-END",
+        "MJD-OBS",
+        "MJD-AVG",
+        "MJD-BEG",
+        "MJD-END",
+    )
```

Comment:
Places like these I have felt free to just make a long list on 2 lines or so and use `# fmt: skip`